subprojects/docs/src/docs/release/notes.md

```diff
@@ -34,22 +34,28 @@ For Java, Groovy, Kotlin and Android compatibility, see the [full compatibility
 <a name="cache"></a>
 ## Gradle's Dependency Cache can be relocated
 
-The Gradle Dependency cache under `$GRADLE_HOME/caches/modules-2` can now be relocated to another directory or host for dependencies cached by Gradle 6.1 and later. When moved to a new location or seeded into a host image, builds using the dependency cache will not need to access the network to download artifacts or metadata if the dependencies have already been downloaded. 
+The strategy of running builds in single use environments in a CI/CD setup is a challenge for build tools as a lot of state that could be reused between builds has to be rebuilt for each execution.
+Gradle has [an advanced dependency cache](userguide/dependency_resolution.html#sec:dependency_cache) which helps reduce downloads of dependencies to a minimum, inside and across build invocations.
+
+With Gradle 6.1, the dependency cache can now be relocated to another directory or host, without the need to preserve absolute paths, for dependencies cached by Gradle 6.1 and later.
```

Comment:
```suggestion
With Gradle 6.1, the dependency cache can now be relocated to another directory or host for dependencies cached by Gradle 6.1 and later.
```

I think this implementation detail is not needed and might be confusing here as it might sound like the use had to do something to (not) preserve something.

Reply:
Agreed 👍